src/dnm.rs

```diff
@@ -195,8 +195,21 @@ impl <'dnmrange> DNMRange <'dnmrange> {
         trimmed_end -= 1; }
       else {
         break; }}
+    if trimmed_start > trimmed_end {  // in case range contains only whitespaces
```

Comment:
Ha, I fixed that independently in my branch, but I set it to the self.end rather than the start. I think I'll merge yours and reintegrate on top.